index.Rmd

```diff
@@ -270,7 +272,7 @@ ggplot(
 
 ```
 
-# Conclusion
+# Conclusion 
```

Comment:
Your findings are really interesting, especially seeing how farm sizes have been impacted by social policies. Hope you can keep developing this project in the future!